tests/Integration/Behaviour/Features/Scenario/Order/order_multi_shop.feature

```diff
@@ -2,6 +2,7 @@
 @reset-database-before-feature
 @clear-cache-before-feature
 @order-multi-shop
+
```

Comment:
Is it necessary? Are the tags still correctly attached to the feature with this?

Reply:
Fixed